currency-map.js

```diff
@@ -362,5 +362,337 @@ module.exports = {
     "TWD": {
         "name": "Taiwan Dollar (TWD)",
         "symbolFormat": "TWD {#}"
+    },
+    "CZK": {
+        "name": "Czech Koruna",
+        "symbolFormat": "CZK {#}"
+    },
+    "FIM": {
+        "name": "",

```

Comment:
The extra currencies after zimbabwe seem to check out. The current answervalues that we should no longer be asking for are:
Lithuanian Lita (LTL)
Latvian Lat (LVL)
Estonian Kroon (EEK)
Slovak Koruna (SKK)
Cyprus Pound (CYP)
Maltese Lira (MTL)
Sudanese Dinar (SDD) - Replaced by Sudanese Pound (SSP) - needs to be added
Surinam Guilder (SRG) - Replaced by Surinam Dollar (SRD) - needs to be added
Reuters (SDP)
Slovenian Tolar (SIT)

The two labeled currencies above need to have new answervalues confirmed before they would be save from the survey. I can confirm the values tomorrow. 

The rest of the currencies have been replaced by the euro or another currency we already have and can just be removed from the available options.